go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus-community/elasticsearch_exporter
 
-go 1.20
+go 1.22.6
```

Comment:
I think with a version this new, the toolchain declaration is required.

Reply:
Please advise on what's needed for the toolchain declaration

Reply:
Do you mean the following declaration?
```
go 1.22.6

toolchain go1.22.6
```
Running `go mod tidy` removes  the `toolchain` line, so I assume it won't be required.

Reply:
I think it depends on your local go version. After a second thought, we don't need to change this file at all. This is only a minimum go version and we don't need any new language features so we can leave this as 1.20

Reply:
I'm go newbie so I had to check, and it appears you're correct. I have go 1.22.5 and when I built the code in master branch and then scan with Syft, it reports that the image contains go standard library 1.22.5. 

Should I remove the change in `go.mod` and rely on CI files to use the updated go version?

Reply:
Done, I removed the commit